### ISSUE TITLE
Export data points Predicted vs. Observed Plot 

### ISSUE THREAD

**Issue by LkKov** [Open-Systems-Pharmacology/OSPSuite.Core#521]
Hello everybody!

Concerning 'observed vs. predicted plots' in ‘Parameter Identifications’: If you right click at the observed vs. predicted plot (see an example attached below) and export the data of this graph, right now you unfortunately do not receive the data of the single observed vs. predicted data points (coloured points) but only the continuous data of all predicted plasma-conc.-time-profile graphs. We would however be interested in the data points of the observed vs. predicted plot. Would it be possible to resolve this issue?

![grafik](https://user-images.githubusercontent.com/49074698/55251011-d458e380-524f-11e9-8c64-ef87122192ab.png)

Thank you very much in advance!

Best regards,
Lukas

**Comment by NinaHanke** [comment]
That would be very helpful!

**Comment by Fatima-Marok** [comment]
Great idea!

**Comment by LMFuhr** [comment]
Yes, this would be great!

**Comment by DeniTue** [comment]
Good idea! 
This would be very helpful.

**Comment by HannahBri** [comment]
Great idea!

**Comment by JGWojtyniak** [comment]
This would be really helpful!

**Comment by msevestre** [comment]
@All: Observed Data will now be exported as a separate table in the excel sheet

**Comment by UlrichSi** [comment]
@msevestre When trying to verify the issue: I noticed that tables with time course data are exported, separately for all simulations and data sets, with differing time steps. I presume what @LkKov really requested was an export of the prediced vs. the observed data, i.e. a 2-column sheet with what is plotted (column1: "predicted", column2: "observed").
The current implementation is better than nothing but in my opinion only slightly better than the previous status.

**Comment by Yuri05** [comment]
s. above

**Comment by msevestre** [comment]
@UlrichSi Fair enough. This is not in the card for this version then. We do not have access to this type of aggregated data at the moment where we export so work needs to be done here.
At any rate, I have fixed a bug where data plotted X vs Y where not exported to Excel :)

**Comment by Fate-afshar** [comment]
greetings everybody!
I've recently faced this issue. has it been resolved?
